app/src/main/java/id/rizmaulana/covid19/ui/dailygraph/DailyDataListFragment.kt

```diff
@@ -0,0 +1,62 @@
+package id.rizmaulana.covid19.ui.dailygraph
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import id.rizmaulana.covid19.databinding.FragmentDailyListBinding
+import id.rizmaulana.covid19.ui.adapter.DailyFactoryImpl
+import id.rizmaulana.covid19.ui.adapter.VisitableRecyclerAdapter
+import id.rizmaulana.covid19.ui.base.BaseFragment
+import id.rizmaulana.covid19.util.ext.observe
+import org.koin.android.viewmodel.ext.android.sharedViewModel
+
+class DailyDataListFragment : BaseFragment() {
+
+    private val viewModel by sharedViewModel<DailyGraphViewModel>()
+
+    private var _binding: FragmentDailyListBinding? = null
+    private val binding get() = _binding!!
+    private val dailyAdapter by lazy {
+        VisitableRecyclerAdapter(DailyFactoryImpl())
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentDailyListBinding.inflate(inflater, container, false)
+        return binding.root
```

Comment:
You can keep it as usual

`return inflater.inflate(R.layout.fragment_daily_list, container, false)`

And do the binding init in `onViewCreated` instead

Reply:
actually i prefer that way because the view is gotten from binding inflation, so i guess it's clear that we are using view binding